### ISSUE TITLE
[FEATURE] Run tests in parallel using a new parallel testing feature

### ISSUE THREAD

**Issue by wwahammy** [houdiniproject/houdini#467]
**Is your feature request related to a problem? Please describe.**
Running tests takes between 5-10 minutes. That's not terrible but it's annoying when you're in the middle of a bunch of work. It'd be nice to have tests run in parallel.

**Describe the solution you'd like**
Running tests in parallel should be easy to set up and not muckup the output by rspec, which is a common problem in in previous testing solutions.

**Describe alternatives you've considered**
We used parallel tests in the past with `parallel_testing` but it really messed with the output to the point of being unusable. We removed that in #466.

**Additional context**
There is work being on on this at https://github.com/ioquatix/turbo_test and https://github.com/rspec/rspec-rails/issues/2104. We should watch that work and then include when they're ready.

**Comment by ilyazub** [comment]
Just replace `parallel_tests` with the https://github.com/serpapi/turbo_tests gem. We at [SerpApi](https://serpapi.com/) use it for six months (since November 2020) in CI and development.

**Comment by wwahammy** [comment]
This is done!
